### PR TITLE
[GSOC][TMVA][SOFIE] Elu ONNX operator implemented

### PR DESCRIPTION
**Pull request by Neel-Shah-29**

### CODE REVIEW

**Comment by lmoneta** [comment]
@phsft-bot build

File: tmva/sofie/inc/TMVA/ROperator_Elu.hxx [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+   ROperator_Elu(float alpha,std::string nameX, std::string nameY):
+   falpha(alpha),fNX(UTILITY::Clean_name(nameX)), fNY(UTILITY::Clean_name(nameY))
+   {
+      if(std::is_same<T, float>::value){
+         fType = "float";
+      }
+		else{
+			throw
+				std::runtime_error("TMVA SOFIE Encountered unsupported type parsing a Elu operator");
+		}
```
Comment (sanjibansg) [inline tmva/sofie/inc/TMVA/ROperator_Elu.hxx:38]: nit: fix the formatting here

**Review by sanjibansg — CHANGES_REQUESTED** [review]
Looks good overall, just a minor change with formatting.

**Review by sanjibansg — APPROVED** [review]
LGTM!

For the future, for coding styles, prefer using a formatting tool like clang-format or style
ref: https://root.cern/contribute/coding_conventions/#astyle

**Pull request merged**